app/slave/cluster_slave.py

```diff
@@ -58,16 +59,13 @@ def api_representation(self):
         """
         executors_representation = [executor.api_representation() for executor in self.executors_by_id.values()]
         return {
-            'connected': self._is_connected(),
+            'is_alive': self.is_alive,
             'master_url': self._master_url,
             'current_build_id': self._current_build_id,
             'slave_id': self._slave_id,
             'executors': executors_representation,
         }
 
-    def _is_connected(self):
-        return self._master_url is not None
```

Comment:
I believe this is no longer used anywhere, and has a slightly overlapping functionality with the self.is_alive attribute.